packages/fleather/lib/src/widgets/editor_toolbar.dart

```diff
@@ -1242,27 +1221,59 @@ class SelectorScope extends StatefulWidget {
 class SelectorScopeState extends State<SelectorScope> {
   OverlayEntry? _overlayEntry;
 
-  Future<T?> pushSelector<T>(Widget selector, Completer<T?> completer) {
+  /// The [context] should belong to the presentor widget.
+  Future<T?> pushSelector<T>(
+      BuildContext context, Widget selector, Completer<T?> completer,
+      {bool rootOverlay = false}) {
```

Comment:
We don't seem to use this option. We should remove it

Reply:
It's a public API and might be used by users implementing their own custom buttons. @amantoux

Reply:
Ok, but still I think we shouldn't offer the option if not required by a user. It adds complexity for no assessed benefit.
You can keep it if you want though, I don't really mind